example/src/main/java/com/transferzero/test/Application.java

```diff
@@ -461,14 +484,15 @@ public static void main(String[] args) throws Exception {
         apiClient.setApiSecret("<secret>");
         apiClient.setBasePath("https://api-sandbox.transferzero.com/v1");
 
-        //accountValidationExample(apiClient);
-        //createTransactionExample(apiClient);
-        //createAndFundTransactionExample(apiClient);
-        //getTransactionByExternalId(apiClient);
-        //getTransactionErrorMessageExample(apiClient);
-        //webhookParseExample(apiClient);
-        //createSender(apiClient);
-        //getSenderByExternalId(apiClient);
-        //updateSender(apiClient);
+//        accountValidationExample(apiClient);
```

Comment:
Can we leave these formatted as before? It's a little neater than putting the comment at the beginning of the line